frontend/src/ui/Access.tsx

```diff
@@ -95,20 +98,20 @@ type AclSelectorProps = {
     ownerDisplayName?: string | null;
     permissionLevels: PermissionLevels;
     addAnonymous?: boolean;
+    itemType?: Item;
```

Comment:
I wouldn't make that optional. The three current use cases seems equally important, randomly setting one (video) as the default seems strange to me.